app/master/build.py

```diff
@@ -564,6 +569,121 @@ def _generate_unique_symlink_path_for_build_repo(self):
         """
         return os.path.join(Configuration['build_symlink_directory'], str(uuid.uuid4()))
 
+    def save(self):
+        """Serialize the Build object and update all of the parts to the database."""
+        session = Connection.get()
+
+        q_build = session.query(BuildSchema).filter(BuildSchema.build_id == self._build_id).first()
```

Comment:
I have no strong opinion about the naming convention here, I'd prefer for the sqlalchemy models to be clearly named though so it's obvious what they are. Open to suggestions!